tlaplus_jupyter/install.py

```diff
@@ -59,14 +71,38 @@ def main(argv=None):
 
     install_my_kernel_spec(user=args.user, prefix=args.prefix)
 
+    # install tla2tools.jar
     vendor_dir = os.path.join(os.path.dirname(__file__), 'vendor')
-    try:
+    if not os.path.isdir(vendor_dir):
         os.mkdir(vendor_dir)
-    except FileExistsError:
-        pass
     jar_path = os.path.join(vendor_dir, 'tla2tools.jar')
     print("Downloading tla2tools.jar to " + jar_path)
     urlretrieve(TOOLS_URI, jar_path)
 
+    # install stats collector id
+    tla_dir = os.path.join(os.path.expanduser("~"), ".tlaplus")
+    if not os.path.isdir(tla_dir):
+        os.mkdir(tla_dir)
+    statfile_path = os.path.join(tla_dir, "esc.txt")
+
+    old_content = None
+    if os.path.isfile(statfile_path):
```

Comment:
I wonder whether this script should touch the file at all if it already exists? At least when `install.py` is called with default arguments that would be unexpected. (Because the user could have deliberately put the file in place / already set up `esc.txt` to disable sharing of statistics or enable anonymous sharing.)

Reply:
Hmm. My intention was that if the user wants to switch off stats collection then he may run install again with some other option set.